stubs/Pillow/@tests/stubtest_allowlist.txt

```diff
@@ -1,5 +1,7 @@
 PIL.FpxImagePlugin
 PIL.ImageQt.ImageQt
-PIL.ImageQt.QImage  # Does exist at runtime usually, but stubtest can't see it
-PIL.ImageQt.QPixmap  # Does exist at runtime usually, but stubtest can't see it
 PIL.MicImagePlugin
+# Requires Pyside or PyQt to be installed.
```

Comment:
Could we add Pyside/PyQt to a `stubtest-requirements.txt` file?

Reply:
We'd have to decide which. It's either/or. Pillow will only use one of the 4.
PySide6 or PyQt6. (PySide2 and PyQt5 being for Qt5).
I haven't looked into details if the signature of the bindings is equivalent. Or if it would matter for stubtest.

PySide is official bindings by the Qt team (aka Qt for Python). Has a public issue tracker.
PyQt is made by riverbank computing, closed source and no issue tracker. More popular because historically (not so much nowadays) it's been more complete than PySide (which didn't always exist). Also the name helps.

Reply:
My instinct would be to go for `PyQt`, purely because I've heard of it (and I'm guessing it's probably more widely used by users of `type s-Pillow` as well) 🙂

Reply:
But I guess we can leave this for now, postpone for another PR